### DISCUSSION TITLE
i dont know if im a idiot or not

### DISCUSSION THREAD

**Discussion by DerpVA** [drizzle-mizzle/CharacterAI-Discord-Bot#146] [Q&A]
I tried following the steps word for word but I still cant manage to make it work. I don't know if I put the token on the wrong place or what i did wrong

**Comment by drizzle-mizzle** [comment]
Well, first, you're on the page of the different project.
Second, don't show me and others your private info like auth tokens.
And third, the problem is that you specify string value (auth token) withoit quotes `"`, just change it to `"your_auth_token"`
